docs/configuration/configuration.md

```diff
@@ -2172,6 +2172,9 @@ it was not set during relabeling. The `__scheme__` and `__metrics_path__` labels
 are set to the scheme and metrics path of the target respectively. The `__param_<name>`
 label is set to the value of the first passed URL parameter called `<name>`.
 
+The `__scrape_interval__` and `__scrape_timeout__` labels are set the target's
```

Comment:
```suggestion
The `__scrape_interval__` and `__scrape_timeout__` labels set the target's
```

Reply:
I believe the proper use is "The `__scrape_interval__` and `__scrape_timeout__` labels are set  _to_ the target's..."

Reply:
yeah